controllers/producersController.js

```diff
@@ -0,0 +1,70 @@
+const Producer = require('../models/producer');
+
+
+exports.producerCreate = function (req,res){
+    //post new producer to the database
+     const producer = new Producer();
+     //body parser lets us use the req.body
+     const {
+         firstName,
+         lastName,
+         phoneNumber,
+         country,
+         state,
+         localGovernment,
+         address,
+         typeOfProducts,
+         createdAt,
+         updatedAt,
+         deletedAt
+      } = req.body;
+      
+     producer.firstName = firstName;
+     producer.lastName = lastName;
+     producer.phoneNumber = phoneNumber;
+     producer.country = country;
+     producer.state = state;
+     producer.localGovernment = localGovernment;
+     producer.addresse = address;
+     producer.typeOfProducts = typeOfProducts;
+     producer.createdAt = createdAt;
+     producer.updatedAt = updatedAt;
+     producer.deletedAt = deletedAt;
```

Comment:
Might be cleaner to use object.assign here. `producer = Object.assign(producer, { firstName, lastName, phoneNumber ...etc};`